security/custom_authentication_provider.rst

```diff
@@ -4,17 +4,13 @@
 How to Create a custom Authentication Provider
```

Comment:
This article is missing a brief introduction explaining when do you need a custom authentication provider. Something like this, but better explained:

```
Symfony provides support for the most :doc:`common authentication mechanisms </security/auth_providers>`. However, your app may need to integrated with some proprietary single-sing-on system or some legacy authentication mechanism. In those cases you need to create a custom authentication provider.
```

Reply:
single-sing-on?, don't you mean single-sign-on 😉 

Unless you are talking about a voice-controlled authentication.